src/Form/Form.js

```diff
@@ -950,15 +950,16 @@ class Form {
         return isLoginOrHybrid && this.device.credentialsImport.isAvailable();
     }
 
-    getFirstViableCredentialsInput() {
-        return [...this.inputs.credentials].find((input) => canBeInteractedWith(input) && isPotentiallyViewable(input));
+    getFirstViableInputForType(dataType) {
+        return [...this.inputs[dataType]].find((input) => canBeInteractedWith(input) && isPotentiallyViewable(input));
     }
 
     async promptLoginIfNeeded() {
```

Comment:
I don't understand why we're doing it here. This method is about autoprompting for login forms (lacks a comment, ouch), whereas the change we're looking at should be done after the initial manual prompt of a credit card prompt.